onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -554,6 +554,7 @@ TensorrtExecutionProvider::TensorrtExecutionProvider(const TensorrtExecutionProv
                                          "TensorRT EP could not open shared library from " + engine_decryption_lib_path_));
     }
     engine_decryption_ = (int (*)(const char*, char*, size_t*))LIBFUNC(handle, "decrypt");
+    engine_encryption_ = (int (*)(const char*, char*, size_t*))LIBFUNC(handle, "encrypt");
```

Comment:
add to VERBOSE log in line 559 below?

Reply:
After discussion with users, we agreed to share the same environment variables as decryption. No new variables are added for encryption.